src/ex1/style.css

```diff
@@ -0,0 +1,192 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
```

Comment:
in general, it's usually a bad practice to use the wildcard selector *, as it applies to all elements, rather than inheriting the style from a parent element (it still could be html or body)

You also have this cool library that normalizes the base/default styles across different browsers - https://necolas.github.io/normalize.css/